app/views/plots/show.html.haml

```diff
@@ -1,10 +1,12 @@
 - content_for_title("Plot ##{@plot.plot_id}")
 - content_for(:subnav) do
-  = link_to '« Plots', plots_path, class: 'btn btn-primary', title: 'List of all plots'
+  = link_to '« Back to Plots', plots_path, class: 'btn btn-primary', title: 'List of all plots'
```

Comment:
Would you mind reverting back to the original? I know it's ugly, but I'm working UI changes in this space and that involves starting to strip out the Bootstrap and changing some wording up.